README.md

```diff
@@ -6,3 +6,9 @@
 
 ## Structure
 ![RCLC Structure with bg](https://user-images.githubusercontent.com/35516367/184010638-25389f58-7de6-4d69-bc0b-5467f6618ef8.png)
+
+## Install
+The `./rclc/` dir needs to be created, so do the two FIFOs. Do this by running the following.
```

Comment:
Comma splice. I believe you meant to say "as do" instead of "so do".
It's `~/.rclc/`, not `./rclc`.
Also, do we need to state what it's doing. IMO it might be better to just not have any description for now, something like:
```md
## Install
`./scripts/install.sh`
## Uninstall
`./scripts/uninstall.sh`
```

Reply:
+1

Reply:
thanks, good catch